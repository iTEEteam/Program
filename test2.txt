loadmap testmap2.txt
buy obstacle 1-1
buy gem 1-1 intensity
make hobbit 1-0
update 4
exit