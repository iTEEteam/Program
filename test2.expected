map testmap2.txt loaded
Obstacle0 created
IntensityGem0 created
Hobbit0 created
update
Hobbit0 moves
Obstacle0 slows Hobbit0 by 1
Obstacle0 worn out
update
update
Hobbit0 moves
update
Hobbit0 reached Mount Doom
VEGE
